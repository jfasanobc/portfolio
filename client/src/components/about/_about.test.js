import React from "react";
import renderer, { create } from "react-test-renderer";
import About from "./about";

describe("About head is rendered? ", () => {
  test("Is has rendered and matching snapshot", () => {
    const aboutBlock = create(<About />);
    expect(aboutBlock.toJSON()).toMatchSnapshot();
  });
});
