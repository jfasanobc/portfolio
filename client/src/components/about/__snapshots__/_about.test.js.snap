// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About head is rendered?  Is has rendered and matching snapshot 1`] = `
<article>
  <p>
    Working towards improving myself, learning new technologies, and advancing my career.
  </p>
</article>
`;
